src/SelectMulti.vue

```diff
@@ -114,7 +114,16 @@
 				open: false,
 				inputValue: ''
 			} as ComponentData
-		},
+        },
+        watch: {
+            loading(newLoadingState, oldLoadingState): void {
+                // In cases where we move from a loading state, to a resolved state...
```

Comment:
in this case, should we also watch `options` and if we get new items notify screen readers ?

Reply:
Yes, this is great @ivanshtyrlyaev ! I'd prefer to leave this PR as is, and not update it here, and make a subsequent PR to add that improvement. Thoughts?

Reply:
> Yes, this is great @ivanshtyrlyaev ! I'd prefer to leave this PR as is, and not update it here, and make a subsequent PR to add that improvement. Thoughts?

Sounds good. Thanks